material/src/main/res/values/attrs.xml

```diff
@@ -6,5 +6,6 @@
         <attr name="materialButtonOutlinedIconStyle" format="reference" />
         <attr name="materialButtonIconStyle" format="reference" />
         <attr name="borderlessButtonIconStyle" format="reference" />
+        <attr name="widgetMaterialButtonStyle" format="reference" />
```

Comment:
This should probably be in its own section as it's got nothing to do with Material Components.

Reply:
Fixed.